src/main/java/main/movie/model/dto/MovieRequestDto.java

```diff
@@ -0,0 +1,30 @@
+package main.movie.model.dto;
+
+public class MovieRequestDto {
+    private String title;
+    private String description;
+
+    public MovieRequestDto(String title, String description) {
+        this.title = title;
+        this.description = description;
+    }
+
+    public MovieRequestDto() {
+    }
```

Comment:
do you use these constructors?

create **MovieResponseDto** as well with `id`, `title` and `description` fields
and use it in your controller in `get()` method